### ISSUE TITLE
[1.15.2] Crash to desktop when trying to open build menu or accept build position

### ISSUE THREAD

**Issue by Drethic** [ldtteam/minecolonies#5492]
# Prerequisites

- [ ] I am running the latest **alpha** version of MineColonies for my Minecraft version.
- [x] I checked the MineColonies [Wiki](https://wiki.minecolonies.ldtteam.com/) and made sure that my issue is not covered there.
- [x] I made sure that **this issue is not a duplicate of any existing issue.**
- [x] This is an issue and not a feature request. Please create all feature requests [here](https://github.com/ldtteam/minecolonies-features/issues).

### Context

- Minecraft Version: 1.15.2
- MineColonies Version: 0.13.86-Alpha
- Structurize Version: 0.13.35-Alpha
- Related Mods and Their Versions: Please see Craft to Exile version 2.1.3b for full list/versions.

### Expected behavior

Building menu opens or build area initializes

### Actual behaviour

While trying to place a new building in an existing village after updating the client crashes to desktop.  We are running in a multiplayer server and different players have experienced the same issue.  Right clicking with the build tool crashes you to desktop either when you first try to right click a block to bring up the interface (user case was right clicking grass block).  I also crashed but not until I hit the accept after positioning the building.  I got the interface to open but I had right clicked short grass.  however, when I hit accept I was facing stone.

### Steps to reproduce the issue

1. Have "Craft to Exile v2.1.3b" mod pack
2. Run game in multiplayer on a server
3. In an existing village made in v2.2.2b try to place any new building
4. Right click with Builder Tool to place the building or accept a building position
5. Crash happens to desktop.  Tested with multiple people on the server.  It does vary if you'll crash when you try to right click or when you accept a build.

### Logs

- crashlog: [Crash Log on Pastebin](https://pastebin.com/LN2ykZNS)

**Comment by Raycoms** [comment]
If you are not " I am running the latest alpha version of MineColonies for my Minecraft version." then you should do that. It will fix it =P

**Comment by Drethic** [comment]
I updated the pack to use minecolonies-0.13.88-Beta and the issue is resolved.  Didn't realize this was fixed in newer versions based on the change log.  Sorry about the reports.
